pkgs/development/python-modules/jschon/default.nix

```
@@ -0,0 +1,72 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, setuptools
+, wheel
+, rfc3986
+, pytestCheckHook
+, hypothesis
+, requests
+, pytest-httpserver
+, pytest-xdist
+}:
+
+buildPythonPackage rec {
+  pname = "jschon";
+  version = "0.11.1";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "marksparkza";
+    repo = "jschon";
+    rev = "v${version}";
+    hash = "sha256-uOvEIEUEILsoLuV5U9AJCQAlT4iHQhsnSt65gfCiW0k=";
+    fetchSubmodules = true;
+  };
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
+  dependencies = [
+    rfc3986
+  ];
+
+  pythonImportsCheck = [
+    "jschon"
+    "jschon.catalog"
+    "jschon.vocabulary"
+    "jschon.exc"
+    "jschon.exceptions"
+    "jschon.formats"
+    "jschon.json"
+    "jschon.jsonpatch"
+    "jschon.jsonpointer"
+    "jschon.jsonschema"
+    "jschon.output"
+    "jschon.uri"
+    "jschon.utils"
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    hypothesis
+    requests
+    pytest-httpserver
+    #pytest-benchmark # not needed for distribution
+    pytest-xdist # not used upstream, but massive speedup
+  ];
+
```

```suggestion
  # remote schema tests need loopback networking access
  __darwinAllowLocalNetworking = true;
```

Alternately, just disabling the problematic tests:
```
  disabledTests = [
    "test_json_loadr"
    "test_remote_source"
  ];
```